### PR TITLE
Added types of databse link, and SQLvsNoSQL

### PR DESCRIPTION
**Pull request by huskypower**
Added a link to types of database on Azure 
and added link to SQLvsNoSQL

### CODE REVIEW

**Comment by huskypower** [comment]
@hadibolehjadi hi bro you have eyes for details , please help to check for mistake , thank you

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Good luck.
   - [What is PaaS?](https://azure.microsoft.com/en-us/overview/what-is-paas/)
   - [What is SaaS?](https://azure.microsoft.com/en-us/overview/what-is-saas/)
   - [Creating an Azure account](https://docs.microsoft.com/en-us/dotnet/azure/create-azure-account)
+  - [Types of Database on Azure](https://azure.microsoft.com/en-us/product-categories/databases/)
+    - [SQL vs NoSQL](https://www.xplenty.com/blog/the-sql-vs-nosql-difference/#:~:text=SQL%20databases%20are%20relational%2C%20NoSQL,dynamic%20schemas%20for%20unstructured%20data.&text=SQL%20databases%20are%20table%20based,graph%20or%20wide%2Dcolumn%20stores.)
```
Comment (ytbryan) [inline README.md]: What is the extra part of the URL? 

is this the actual URL? https://www.xplenty.com/blog/the-sql-vs-nosql-difference/

File: README.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Good luck.
   - [What is PaaS?](https://azure.microsoft.com/en-us/overview/what-is-paas/)
   - [What is SaaS?](https://azure.microsoft.com/en-us/overview/what-is-saas/)
   - [Creating an Azure account](https://docs.microsoft.com/en-us/dotnet/azure/create-azure-account)
+  - [Types of Database on Azure](https://azure.microsoft.com/en-us/product-categories/databases/)
```
Comment (ytbryan) [inline README.md]: "Types of Databases on Azure"

File: README.md (hunk 1 above)
Comment (huskypower) [reply README.md]: This is the actual URL that you have pointed out https://www.xplenty.com/blog/the-sql-vs-nosql-difference/.
will make the changes, thank you for the review

File: README.md (hunk 2 above)
Comment (huskypower) [reply README.md]: Noted on the 's' on database, and will do the changes, thank you for the review

**Pull request merged**